<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleaseChangelogDialog</class>
 <widget class="QDialog" name="ReleaseChangelogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Changelog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <item>
    <layout class="QGridLayout" name="headerLayout" columnstretch="0,1">
     <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignBottom">
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="platformLabel">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2" alignment="Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="iconLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="changelogBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
